xc7/utils/prjxray_create_edges.py

```diff
@@ -485,35 +482,32 @@ def make_connection(conn, input_only_nodes, output_only_nodes,
 
     """
 
-    src_wire_pkey, tile_pkey, src_node_pkey = find_wire(tile_name, tile_type, pip.net_from)
-    sink_wire_pkey, tile_pkey2, sink_node_pkey = find_wire(tile_name, tile_type, pip.net_to)
+    c = conn.cursor()
 
-    assert tile_pkey == tile_pkey2
+    # Get node pkeys
+    src_node_pkey = c.execute("SELECT node_pkey FROM wire WHERE tile_pkey = (?) AND wire_in_tile_pkey = (?)", (tile_pkey, src_wire_pkey)).fetchone()[0]
```

Comment:
Probably worth making a utility method for "get_node_from_wire"

Reply:
As you suggested I added a method "get_node_from_wire"